third-party/pipelines-as-code/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@kubernetes-models/pipelines-as-code",
+  "version": "1.0.0",
```

Just leave it `0.0.0` and changeset will bump the version automatically.

```suggestion
  "version": "0.0.0",
```